Directory.Build.props

```diff
@@ -67,8 +67,12 @@
     <HostArch>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant)</HostArch>
     <ArchGroup Condition="'$(ArchGroup)' == '' AND '$(HostArch)' == 'arm'">arm</ArchGroup>
     <ArchGroup Condition="'$(ArchGroup)' == '' AND '$(HostArch)' == 'arm64'">arm64</ArchGroup>
+    <ArchGroup Condition="'$(ArchGroup)' == '' AND '$(OSGroup)' == 'WebAssembly'">wasm</ArchGroup>
     <ArchGroup Condition="'$(ArchGroup)' == ''">x64</ArchGroup>
 
+    <!-- Runtime OS is calculated based on the build system OS, however if building for WebAssembly we need to let the build system that. -->
```

Comment:
Nit: "we need to let the build system that" is missing a word

Reply:
Thanks, will update.